lib/Driver/ToolChains/Flang.cpp

```diff
@@ -38,6 +42,23 @@ using namespace clang::driver::tools;
 using namespace clang;
 using namespace llvm::opt;
 
+class FlangMacroBuilder : public MacroBuilder {
+  ArgStringList &CmdArgs;
+  const ArgList &Args;
+  public:
+    FlangMacroBuilder(ArgStringList &UpperCmdArgs, const ArgList &Args, llvm::raw_string_ostream &Output)
```

Comment:
Maybe use a different name for  the parameter `Args` here?

Reply:
Any suggestions? I used `UpperCmdArgs, Args` because they were the name in the code where `FlangMacroBuilder` is called.

Reply:
`IncomingArgs` or `DriverArgs` ?